### PR TITLE
fix: wishlist and trailer button margin/padding issues

### PR DESCRIPTION
**Pull request by PeculiarGoat**
https://dev.azure.com/S72/SHIFT72/_workitems/edit/5098

- removed padding from left of wishlist button so it aligns to left of container when trailer button is missing.
- added margin to right of trailer button so wishlist button isn't too close.
- removed margin and padding from wishlist button on posters so it aligns to the center.

### CODE REVIEW

File: site/styles/_buttons.scss [hunk 1]
```diff
@@ -124,6 +124,14 @@
       fill: #000;
       stroke: #000;
     }
+  }
+  &.s72-btn-trailer {
```
Comment (stajs) [inline site/styles/_buttons.scss:128]:
```suggestion
  }
  
  &.s72-btn-trailer {
```

File: site/styles/_buttons.scss [hunk 2]
```diff
@@ -124,6 +124,14 @@
...
+    background: transparent;
+    border: 0 none;
```
Comment (stajs) [inline site/styles/_buttons.scss:130]: Curious what the `none` is doing? I thought `0` would suppress it?

**Review by stajs — APPROVED** [review]
Minor Q.

**Review by imran-salim — APPROVED** [review]

File: site/styles/_buttons.scss (hunk 2 above)
Comment (PeculiarGoat) [reply site/styles/_buttons.scss:131]: That's been merged in from master... I didn't write it/don't know what it's for.

**Pull request merged**